main.js

```diff
@@ -10,6 +13,27 @@ document.addEventListener('DOMContentLoaded', createPrompt(letters))
 
 var $firstChild = $pangram.children[0]
 
+//make first array item active
```

Comment:
You can remove comments. Try to rely on good names instead. Comments are only warranted for extremely long/confusing algorithms.